build-system/jscodeshift/README.md

```diff
@@ -0,0 +1,7 @@
+# [jscodeshift](https://github.com/facebook/jscodeshift)
+
+This directory holds shared configuration and utils for **jscodeshift**.
```

Comment:
I don't think this should be a top-level directory in `build-system/`. Since it has to do with configuration, can we move the entire `jscodeshift/` directory to `build-system/test-configs/jscodeshift/`?